src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/projects/RemoteProjectInfo.java

```diff
@@ -0,0 +1,76 @@
+package ca.corefacility.bioinformatics.irida.ria.web.ajax.projects;
+
+import java.util.Date;
+
+import ca.corefacility.bioinformatics.irida.model.RemoteAPI;
+import ca.corefacility.bioinformatics.irida.model.project.ProjectSyncFrequency;
+import ca.corefacility.bioinformatics.irida.model.remote.RemoteStatus;
+import ca.corefacility.bioinformatics.irida.model.user.User;
+
+public class RemoteProjectInfo {
```

Comment:
We should come up with a naming scheme for these, we have models, details, etc...

Reply:
Agreed

Reply:
Let's go with *Settings as that's what this dto is and we can use this naming scheme going forward

Reply:
Updated in d6e05c3fc8762cb07c7bd654025517140657c154